### PR TITLE
iOS: set number of lines on Label

### PR DESCRIPTION
**Pull request by adly-holler**
@joshaber

### CODE REVIEW

File: Few-iOS/Label.swift [hunk 1]
```diff
@@ -9,32 +9,36 @@
...
-private let StringFudge = CGSize(width: 4, height: 0)
 
 private let ABigDimension: CGFloat = 10000
 
-internal func estimateStringSize(string: NSAttributedString, maxSize: CGSize = CGSize(width: ABigDimension, height: ABigDimension)) -> CGSize {
-	let rect = string.boundingRectWithSize(maxSize, options: .UsesLineFragmentOrigin | .UsesFontLeading, context: nil)
-	let width = ceil(rect.size.width) + StringFudge.width
-	let height = ceil(rect.size.height) + StringFudge.height
-	return CGSize(width: width, height: height)
+internal let sizingLabel = UILabel()
```
Comment (joshaber) [inline Few-iOS/Label.swift]: This can probably be private right?

**Comment by joshaber** [comment]
Awesome, just :one: :wrench:

**Comment by adly-holler** [comment]
Ah! Good point. Updated and back at you @joshaber

**Comment by joshaber** [comment]
:metal:

**Pull request merged**